car-bsp/DataModules/src/Mppt.cpp

```diff
@@ -38,7 +38,7 @@ void Mpptx0::ToByteArray(uint8_t* buff) const
 	}
 	f2b.f = inputCurrent;
 	for (int i=4;i<=7;i++){
-		buff[i] = f2b.s[i];
+		buff[i] = f2b.s[i - 4];
```

Comment:
Cool with it b/c it fixes the bug, but we should update this to use memcpy which will be much more efficient since in its current implementation the loops can't be optimized out by the compiler.